windows.md

```diff
@@ -390,6 +390,49 @@ With those compatibility things out of the way, you're ready to start the system
     4. Open the start menu and search for "Ubuntu". Start it - it should ask you to create a user with a password. This will be your user to log in to your Ubuntu Linux Subsystem - note down the username and password somewhere secure to make sure you do not forget it.
     5. Open the start menu and search for "Docker Desktop". Start it and go to the Settings. Under the General tab, you will find an option called "Use WSL 2 based engine". Make sure this is checked.
 
+    If opening Docker Desktop from **Option A or B** shows an error, you may need to enable virtualization on your machine
+
+    The error might look like this:<br><br>
+    <img src="docker-desktop-opening-error.avif"><br><br>
+
+    Close PowerShell and open it again as administrator (like in step 2)
+
+    Check if your computer supports virtualization. Run this command:
```

Comment:
this should follow the style of the other instructions to enter commands in the guide

Reply:
> is it possible to limit the number of properties being displayed? are all 5 really relevant?

All five are not necessarily relevant, but I just wanted to keep the expected output the way it is so it doesn't feel confused

https://github.com/upleveled/system-setup/pull/16/commits/493e54b7da21449f98e031dc87aead474b1a121d
https://github.com/upleveled/system-setup/pull/16/commits/63fe30ab3bddc4701ef0e54ac5d1981c9ee84d24

Reply:
> All five are not necessarily relevant, but I just wanted to keep the expected output the way it is so it doesn't feel confused

I agree that we should display the output exactly as it appears on the screen for students, but I think we can probably limit the number of lines of output to make it easier for students to check each line

eg. [piping to `Select-Object` with commas: `Get-ComputerInfo -Property * | Select-Object CsName,CsModel`](https://community.spiceworks.com/t/get-computerinfo-cmdlet-all-servers/712473)

Reply:
The command has been improved

https://github.com/upleveled/system-setup/commit/2465d51de575c0d7a4b0bbd1b307117bfab332cb